docs/installation/airgap.md

```diff
@@ -43,7 +44,7 @@ This method requires you to manually deploy the necessary images to each node, a
 ### Embedded Registry Mirror
 
 :::info Version Gate
-The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1
+The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1 and GA as of December 2024 releases: v1.29.12+k3s1, v1.30.8+k3s1, v1.31.4+k3s1
 :::
 
 K3s includes an embedded distributed OCI-compliant registry mirror.
```

Comment:
Can we just shorten this section and point to actual docs.

Reply:
Sorry, what docs do you mean?